pkg/sqlcmd/errors.go

```diff
@@ -73,23 +74,24 @@ func (e *VariableError) IsSqlcmdErr() bool {
 func ReadOnlyVariable(variable string) *VariableError {
 	return &VariableError{
 		Variable:      variable,
-		MessageFormat: "The scripting variable: '%s' is read-only",
+		MessageFormat: localizer.Sprintf("The scripting variable: '%s' is read-only", variable),
```

Comment:
>MessageFormat [](http://example.com/codeflow?start=2&length=13)

technically this is a breaking change but I doubt anything external is depending on the current behavior.
It's probably worth having a note in the merge commit message though.

Reply:
Yeah, I tried multiple things to keep the previous approach but then the scanner failed to recognize what '%s' should be replaced with and couldn't replace %s with anything when actual localized error message was produced.